### PR TITLE
Opls, convergence and opt.xyz files

### PR DESCRIPTION
**Pull request by jthorton**

### CODE REVIEW

File: geometric/engine.py [hunk 1]
```diff
@@ -379,6 +384,48 @@ def calc_new(self, coords, dirname):
...
+        for index in range(nonbonded_force.getNumParticles()):
+            charge, sigma, epsilon = nonbonded_force.getParticleParameters(index)
+            # print(nonbonded_force.getParticleParameters(index))
+            ljset[index] = (sigma, epsilon)
+            lorentz.addParticle([sigma, epsilon])
+            nonbonded_force.setParticleParameters(
+                index, charge, 0, 0)
+        for i in range(nonbonded_force.getNumExceptions()):
+            (p1, p2, q, sig, eps) = nonbonded_force.getExceptionParameters(i)
+            # ALL THE 12,13 and 14 interactions are EXCLUDED FROM CUSTOM NONBONDED
```
Comment (proteneer) [inline geometric/engine.py]: Does this ignore the typical 0.5 scaling factor using for 1-4 interactions?

File: geometric/engine.py [hunk 2]
```diff
@@ -379,6 +384,48 @@ def calc_new(self, coords, dirname):
...
+            # FORCE
+            lorentz.addExclusion(p1, p2)
+            if eps._value != 0.0:
+                sig14 = sqrt(ljset[p1][0] * ljset[p2][0])
+                eps14 = sqrt(ljset[p1][1] * ljset[p2][1])
```
Comment (jthorton) [inline geometric/engine.py]: Actually, I don't think we need to calculate eps14 again we should just return eps back to the exception parameters so we don't interfere with the value.

File: geometric/engine.py (hunk 1 above)
Comment (jthorton) [reply geometric/engine.py]: No, it is used we state it in the FF XML file and are just changing how the sigma values are combined here, I think this comment needs updating.

Comment (jthorton) [reply geometric/engine.py]: No problem I think ill change this and resubmit the pull request?

**Pull request closed without merging**